bindings/python/native/tests/fixtures/test_vectors.json

```diff
@@ -31,7 +31,7 @@
         ]
     },
     "UTXOINPUT": [
-        "7702ea0f2cd6af3206b894c3f2fe4362b23f0f4828857d31e733103b09db25840000"
+        "00000000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
What's the purpose for this change here? @Thoralf-M

Reply:
The other output exists only in the old testnet and the nodes of the new devnet don't know about it, but about the genesis output

Reply:
Ok apparently not anymore, but I need funds first so I can create a new output that will not be moved